test/e2e/tests/custom-token-add-approve.spec.js

```diff
@@ -254,12 +254,12 @@ describe('Create token, approve token and approve token without gas', function (
         );
 
         // editing gas fee
-        const editBtn = await driver.findElements({
+        const editBtn = await driver.findElement({
```

Comment:
we only have one element with text 'Edit' and tag h6, no need to use findElements.